drivers/Aqara/aqara-lock/src/test/test_aqara_lock.lua

```diff
@@ -0,0 +1,153 @@
+local test = require "integration_test"
+local t_utils = require "integration_test.utils"
+local capabilities = require "st.capabilities"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+local zigbee_test_utils = require "integration_test.zigbee_test_utils"
+
+local remoteControlStatus = capabilities.remoteControlStatus
+local lockCredentialInfo = capabilities["stse.lockCredentialInfo"]
+test.add_package_capability("lockCredentialInfo.yaml")
+local Battery = capabilities.battery
+local Lock = capabilities.lock
+local TamperAlert = capabilities.tamperAlert
+
+local PRI_CLU = 0xFCC0
+local PRI_ATTR = 0xFFF3
+local MFG_CODE = 0x115F
```

Comment:
duplicate constants, can we get these from the driver somehow?

Reply:
The constants is not duplicated in the "test_aqara_lock.lua" file.
Please explain the problem in detail.